src/index.js

```diff
@@ -1,34 +1,28 @@
-import './style.css';
+/* eslint-disable import/no-unresolved */
+/* eslint-disable import/no-cycle */
 
-const tasks = [{
-  description: 'Coding milestone projects',
-  completed: false,
-  index: 1,
-},
-{
-  description: 'Learning JS ES6',
-  completed: false,
-  index: 2,
-},
-{
-  description: 'Attending morning session meeting',
-  completed: false,
-  index: 3,
-},
-{
-  description: 'Going for lunch break',
-  completed: false,
-  index: 4,
-},
-];
+import './style.css';
+import editTask from './modules/editInput.js';
+import TaskList from './modules/render.js';
+import addTask from './modules/addTask.js';
+// import RemoveTask from './modules/deleteTask.js';
```

Comment:
- [ ] As you have done an excellent job creating the modules :+1: , it would be beneficial if you remove the commented-out code from your files. This will improve the readability and maintainability of the codebase, making it easier for other developers to understand and modify the code if necessary. Additionally, removing unused code can also help to reduce the size of the files, which can lead to faster load times and improved performance :100: .